andela/AndelaEatsApi
# #164679973 Automatically create  meal sessions

**What does this PR do?**
* Creates a Cron job to automatically create meal sessions for each location at midnight.

**Description of Task to be completed?**
* Create a Cron job to run at midnight
* Write tests for functionality
* Refactor meal-session logic into a separate business logic package

**How should this be manually tested?**
* Pull and checkout to this branch locally by running `git fetch origin ft-auto-create-meal-sessions-164679973 && git checkout ft-auto-create-meal-sessions-164679973`
* Change the time on your computer to 23:59:00 while also setting the time zone to Africa/Lagos
* Run the command `flask run --no-reload` to ensure that flask does not spin up two instances of itself.
* Wait for your local time to reach 00:00:00.
* Using pgAdmin or any other database browsing tool, check the meal_sessions table; you should be able to find newly created meal sessions.

**Any background context you want to provide?**
 N/A

**What are the relevant pivotal tracker stories?**
[164679973](https://www.pivotaltracker.com/story/show/164679973)
